src/scaffoldmaker/utils/vector.py

```diff
@@ -42,3 +42,50 @@ def addVectors(v1,v2,s1=1.0,s2=1.0):
     :return: Vector s1*v1+s2*v2
     '''
     return [(s1 * v1[c] + s2 * v2[c]) for c in range(len(v1))]
+
+
+def scalarProjectionOfV1OnV2(v1, v2):
```

Comment:
Rename to just scalarProjection. Same for following 2 functions, remove OfV1OnV2.